crates/neon-runtime/src/napi/call.rs

```diff
@@ -61,6 +96,7 @@ pub unsafe fn data(env: Env, info: FunctionCallbackInfo, out: &mut *mut c_void)
 }
 
 /// Gets the number of arguments passed to the function.
+// TODO: Remove this when `CallContext` is refactored to get call info upfront.
```

Comment:
Idea: would it be useful to label TODO comments with labels for the future trigger that will allow the action to be taken, e.g. `TODO(remove-legacy)`?